brave/brave-core
# (Wallet) Improve Wallet dark theme

## Resolves https://github.com/brave/brave-browser/issues/33439 

## Preview

| Dark Mode    | Light Mode |
| --------- | ------- |
|    <img width="489" alt="dark" src="https://github.com/brave/brave-core/assets/3308503/b777df3d-239c-45f7-bfdc-e574cc8c2da4">  |      <img width="489" alt="light" src="https://github.com/brave/brave-core/assets/3308503/14638cd1-775a-4316-a368-fde0fae77ef4">   |



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
- Select dark mode from Settings > Appearance > Theme.
- Open Brave Wallet.
- Verify toolbar background color matches Figma design.
- Verify toolbar does not have a shadow but a thin divider line.
- Verify Wallet background color matches Figma design.
- Verify selector (Assets - NFTs) main color and background color match Figma design.
- Verify bottom navigation view colors and text size match Figma design.
- Verify a thin divider line is shown above bottom navigation view.
